src/pages/home/report/ReportActionCompose/ComposerWithSuggestions.js

```diff
@@ -185,6 +185,12 @@ function ComposerWithSuggestions({
             setIsCommentEmpty(!!newComment.match(/^(\s)*$/));
             setValue(newComment);
             if (commentValue !== newComment) {
+                // Reset emoji suggestions when an emoji is replaced.
+                // Important when emoji is replaced and the selection is not changed, more info issue #27156
```

Comment:
```suggestion
                // Ensure emoji suggestions are hidden even when the selection is not changed (so calculateEmojiSuggestion would not be called).
```
I think we can make this comment a bit more concise.

Reply:
updated